scripts/install-server.sh

```diff
@@ -4,9 +4,9 @@ set -e
 
 readonly ARGS=("$@")
 readonly PROGNAME="./install-server.sh"
-TRENTO_SERVER_CHART_VERSION=${TRENTO_SERVER_CHART_VERSION:-"2.4.0"}
+TRENTO_SERVER_CHART_VERSION=${TRENTO_SERVER_CHART_VERSION:-"2.3.2"}
```

Comment:
I guess as we have currently 2.3.2 released right?

Reply:
Yes. We must not change this install-server script except if we are creating a new release, which was not the case